frontera/contrib/backends/remote/codecs/msgpack.py

```diff
@@ -65,33 +68,34 @@ def __init__(self, request_model, response_model, *a, **kw):
         self._response_model = response_model
 
     def _response_from_object(self, obj):
-        return self._response_model(url=obj[0],
+        url = to_native_str(obj[0])
```

Comment:
Do you think it is better to call `to_native_str` twice? Once for response_model and then for request_model inside it.